.ci/bump-releasenotes-builder-version.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
```

Comment:
name of script should be `bump-cs-version-in-releasenotes-builder.sh`
please apply the same naming to all other shell files of such type that you are introducing.

Reply:
script should take version as argument to let Github action take it ones from metadata file and all scripts just update version without going to internet.

Reply:
Changes Done